mods/ra/rules/player.yaml

```diff
@@ -95,6 +95,10 @@ Player:
 	DeveloperMode:
 		CheckboxDisplayOrder: 10
 	GpsWatcher:
+		RequiresCondition: gps-online
```

Comment:
So if I read this correctly a player's watcher trait is only active when this condition is enabled by having the `gps-uplink` prerequisite which means that you need your own tech lab to use an allies' gps?

Reply:
No, `GrantedAllies` only checks if the allied granted GPS is online not yours.

Reply:
I see. This is a bit confusing.

Reply:
To be honest, I haven't tested this case.

Reply:
Implementations of `RequiresCondition` / `IsTraitDisabled` are required to behave as if the trait were not defined when disabled. It sounds like this isn't actually disabling the trait, so it can't use the standard conditional plumbing.

You can use the lower level `IObservesVariables` plumbing to implement this as a `UseAllyGPSCondition` field instead.